server/service/integration_enterprise_test.go

```diff
@@ -4659,6 +4648,59 @@ func (s *integrationEnterpriseTestSuite) TestRunHostSavedScript() {
 	require.False(t, scriptResultResp.HostTimeout)
 	require.Contains(t, scriptResultResp.Message, fleet.RunScriptAlreadyRunningErrMsg)
 	require.Nil(t, scriptResultResp.ScriptID)
+
+	// Verify that we can't enqueue more than 1k scripts
+
+	// Make the host offline so that scripts enqueue
+	err = s.ds.MarkHostsSeen(ctx, []uint{host.ID}, time.Now().Add(-time.Hour))
+	require.NoError(t, err)
+	for i := 0; i < 1000; i++ {
```

Comment:
If this is taking significant time (it probably is, especially on slow CI?), it'd be best to alter the limit for the test (e.g. 5 or 10).

Reply:
Yeah, I originally was hitting the actual endpoint to make the new script execution and that took a long time even on my workstation. Looks like the CI runs are taking about the same time as before? https://github.com/fleetdm/fleet/actions/runs/7613605806/job/20733884798?pr=16183

Reply:
:+1: allright sounds good if it's barely noticeable.